app/controllers/home_controller.rb

```diff
@@ -1,4 +1,19 @@
-class HomeController < ApplicationController
-  def index
-  end
-end
+class HomeController < ApplicationController
+  def index
+  end
+
+  def about

```

Comment:
Are we sure all these functions belong in the home controller?  I'm not convinced this is the right place for them.

Reply:
I mostly put these here to auto-generate the correct view. Not sure how else to do it tbh. I was thinking of "home" as in the bail website in general, and "donate" and "apply" as different parts.

Reply:
It's an interesting thing because they don't need their own controller per se because they don't have new models, but I would argue it's better practice to make new controllers anyway.  There's a single command line thing you can run that auto-generates all the new files and updates the routes and everything.  I believe it is "rails generate ControllerName actionName".

Reply:
My issue with this is that then we'd have like 10 controllers. Each controller would basically only have **one** action/view, which would be to display the correct page, which are mostly static/text. I think it'd just unnecessarily clutter the directory. If you want we can rename the ```Home``` controller to something like ```Info```.

Reply:
I'm ok with renaming it.

Reply:
renamed to ```Info```